### PR TITLE
Upd/jsdoc examples

### PR DESCRIPTION
**Pull request by edgarsilva**

### CODE REVIEW

File: lib/sphero.js [hunk 1]
```diff
@@ -24,6 +24,16 @@ function classCallCheck(instance, Constructor) {
...
 }
 
+/**
+ * Creates a new sphero instance
+ * @constructor
+ * @param {String} address of the connected sphero
+ * @param {Object} opts for sphero setup
+ * @example
+ * var sphero = require("sphero");
+ * var spheroRPB = sphero("/dev/rfcomm0", { timeout: 300 });
```
Comment (deadprogram) [inline lib/sphero.js]: Is there a place we describe the typical formats of port naming for OSX, Windows, and Linux? Having the Linux port as the example is probably the smallest audience of users, I suggest using OSX as default.

Comment (edgarsilva) [inline lib/sphero.js]: @stewart can you help with that? for windows it would be just `COM1` `COM2` etc...

Comment (edgarsilva) [inline lib/sphero.js]: Actually we can add multiple examples adding now for windows, @stewart can then help with OSX.

File: lib/sphero.js [hunk 2]
```diff
@@ -49,6 +59,36 @@ var Sphero = module.exports = function Sphero(address, opts) {
 
 util.inherits(Sphero, EventEmitter);
 
+/**
+ * Stablishes a connection to the sphero, this allows to
```
Comment (zankich) [inline lib/sphero.js]: This should be `Establishes`

File: lib/sphero.js [hunk 3]
```diff
@@ -24,6 +24,16 @@ function classCallCheck(instance, Constructor) {
...
 
+/**
+ * Creates a new sphero instance
+ * @constructor
+ * @param {String} address of the connected sphero
+ * @param {Object} opts for sphero setup
+ * @example
+ * var sphero = require("sphero");
+ * var spheroRPB = sphero("/dev/rfcomm0", { timeout: 300 });
+ * @return {void}
```
Comment (zankich) [inline lib/sphero.js]: This does not return `void` it returns a `Sphero`

Comment (edgarsilva) [inline lib/sphero.js]: returns void, constructors are not called on their own, need to be instantiated with `new`.

Comment (edgarsilva) [inline lib/sphero.js]: Well it is confusing, :joy: I will upd though.

Comment (stewart) [inline lib/sphero.js]: Marking as a `@constructor` indicates it's supposed to be called with `new`, anyways, so a non-`void` `@return` signature seems warranted.

File: lib/sphero.js (hunk 1 above)
Comment (stewart) [inline lib/sphero.js]: An example OS X port looks like `cu.Sphero-OGB-AMP-SPP` or `tty.Sphero-OGB-AMP-SPP`.

**Pull request closed without merging**